doc/source/parameter.rst

```diff
@@ -0,0 +1,90 @@
+=============
+``/parameter``
+=============
+
+.. http:get:: /parameter/<parameter_id>
+
+    Get data about the ``<parameter_id>`` parameter
```

Comment:
“data about” could be inferred with the name of the endpoint. “all information contained in the loaded tax and benefit system about” at least tells me that I will get a full dump. Maybe “the type, description and values”?